python-lib/sharepoint_client.py

```diff
@@ -370,6 +389,9 @@ def get_add_folder_url(self, full_path):
     def get_file_add_url(self, full_path, file_name):
         return self.get_folder_url(full_path) + "/Files/add(url='{}',overwrite=true)".format(file_name)
 
+    def get_list_default_view_url(self, list_title):
+        return self.get_lists_by_title_url(list_title) + "/DefaultView/ViewFields"
```

Comment:
- I personally usually use os.path.join to join urls as it handles slashs misplacement
```suggestion
    def get_list_default_view_url(self, list_title):
        return os.path.join(self.get_lists_by_title_url(list_title), "DefaultView/ViewFields")
```
- Maybe it would be cleaner to place this endpoint to a constant?